modules/Attendance/report_courseClassesNotRegistered_byDate.php

```diff
@@ -194,7 +195,7 @@
                     echo $row['courseShort'].'.'.$row['class'];
                     echo '</td>';
                     echo '<td>';
-                    echo date('M j', $timestampStart).' - '. date('M j, Y', $timestampEnd);
+                    echo Format::dateReadable($dateStart, '%b %d').' - '. Format::dateReadable($dateEnd, '%b %d, %Y');
```

Comment:
Check out the dateRangeReadable method here, it automatically handles whether the month & year should be shown for date ranges eg: Jan 5-15, 2019 vs Jan 5 - Feb 5, 2019, vs Dec 20 2018 - Jan 5, 2019

Reply:
That's great!